src/status.rs

```diff
@@ -453,14 +455,16 @@ impl StatusBuilder {
     }
 
     pub fn build(self) -> Result<Status, String> {
-        Ok(Status::new(
-            self.space,
-            self.logo.ok_or("logo missing")?,
-            self.url.ok_or("url missing")?,
-            self.location.ok_or("location missing")?,
-            self.contact.ok_or("contact missing")?,
-            self.issue_report_channels,
-            ))
+        Ok(Status {
+            api: "0.13".into(), // TODO: Deduplicate
```

Comment:
Sorry, I didn't mean the validator. Rather, I meant the bot.

But the point about supported versions is valid. Can you create a separate issue for this?